packages/app-project/src/screens/ProjectHomePage/components/Hero/components/Background/Background.js

```diff
@@ -0,0 +1,40 @@
+import { withResponsiveContext } from '@zooniverse/react-components'
+import { string } from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+const Img = styled.img`
+  height: 100%;
+  min-height: 61.8vh;
```

Comment:
We might want a max height (cc @beckyrother)? It looks a bit big on iPad Pro sizes: 

<img width="617" alt="Screen Shot 2019-09-03 at 2 35 10 PM" src="https://user-images.githubusercontent.com/5016731/64202958-0f42f100-ce58-11e9-97b7-5439d4647272.png">

Reply:
I didn't realize y'all were working on this page!!!!! Really excited, cried a little, this looks great. 

Yes, you're right that there should be a max height. Here's an example where the content is quite tall; it's the max height of the hero image and list of workflows before the workflow list starts to scroll (this should never happen but just in case). The height of the hero image is **763px**. https://projects.invisionapp.com/d/main#/console/12924056/347708839/inspect

There's also a mobile version: https://projects.invisionapp.com/d/main#/console/12924056/344288107/inspect

Reply:
Why is the minimum height ~62% of the viewport height? That seems big to me, particularly in portrait layouts. For images, I'd set `min-height` in pixels.

Reply:
You could also use [`align-items: stretch`](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items) if you want the image height to match the height of the box next to it.

Reply:
The most important part of this page is the workflow selection, which is in the module next to the hero image. Thus, the hero image should take up this amount of the screen. For portrait layouts, please use the mobile version attached above.

Reply:
okay, `max-height` is now 763px on desktop and height is derived from the workflow select box